package.json

```diff
@@ -6,24 +6,25 @@
   "author": "Orta Therox",
   "license": "MIT",
   "scripts": {
-    "build": "tsc",
+    "build": "yarn tsc",
+    "postinstall": "yarn build",
     "test": "jest",
     "dev": "concurrently --kill-others \"tsc -w\" \"nodemon dist/index.js\"",
     "start": "node dist/index.js"
   },
   "dependencies": {
+    "@types/express": "^4.11.1",
+    "@types/node-fetch": "^1.6.7",
     "express": "^4.15.3",
-    "node-fetch": "^1.7.1"
+    "node-fetch": "^1.7.1",
+    "typescript": "^2.7.1"
```

Comment:
let's heroku compile the app during the build process:

```sh
~/dev/projects/artsy/js/apps/gh-commentify fix_heroku

❯ git push heroku fix_heroku:master
Counting objects: 8, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (8/8), done.
Writing objects: 100% (8/8), 2.05 KiB | 2.05 MiB/s, done.
Total 8 (delta 5), reused 0 (delta 0)
remote: Compressing source files... done.
remote: Building source:
remote:
remote: -----> Node.js app detected
remote:
remote: -----> Creating runtime environment
remote:
remote:        NPM_CONFIG_LOGLEVEL=error
remote:        NPM_CONFIG_PRODUCTION=true
remote:        NODE_VERBOSE=false
remote:        NODE_ENV=production
remote:        NODE_MODULES_CACHE=true
remote:
remote: -----> Installing binaries
remote:        engines.node (package.json):  unspecified
remote:        engines.npm (package.json):   unspecified (use default)
remote:        engines.yarn (package.json):  unspecified (use default)
remote:
remote:        Resolving node version 8.x...
remote:        Downloading and installing node 8.9.4...
remote:        Using default npm version: 5.6.0
remote:        Resolving yarn version 1.x...
remote:        Downloading and installing yarn (1.4.0)...
remote:        Installed yarn 1.4.0
remote:
remote: -----> Restoring cache
remote:        Loading 2 from cacheDirectories (default):
remote:        - node_modules
remote:        - bower_components (not cached - skipping)
remote:
remote: -----> Building dependencies
remote:        Installing node modules (yarn.lock)
remote:        yarn install v1.4.0
remote:        [1/4] Resolving packages...
remote:        success Already up-to-date.
remote:        $ yarn build
remote:        yarn run v1.4.0
remote:        $ yarn tsc
remote:        $ /tmp/build_15087e6e6c37db1e1edeacbac833c9aa/node_modules/.bin/tsc
remote:        Done in 2.69s.
remote:        Done in 3.94s.
remote:
remote: -----> Caching build
remote:        Clearing previous node cache
remote:        Saving 2 cacheDirectories (default):
remote:        - node_modules
remote:        - bower_components (nothing to cache)
remote:
remote: -----> Build succeeded!
remote: -----> Discovering process types
remote:        Procfile declares types     -> (none)
remote:        Default types for buildpack -> web
remote:
remote: -----> Compressing...
remote:        Done: 24.7M
remote: -----> Launching...
remote:        Released v11
remote:        https://artsy-blog-gh-commentify.herokuapp.com/ deployed to Heroku
remote:
remote: Verifying deploy... done.
To https://git.heroku.com/artsy-blog-gh-commentify.git
   82a8c50..27b9ea4  fix_heroku -> master
```

Reply:
Good idea 👍